lib/console_web/schema/schema.ex

```diff
@@ -135,5 +135,29 @@ defmodule ConsoleWeb.Schema do
         {:ok, topic: "#{args.context_name}/#{args.context_id}"}
       end
     end
+
+    field :device_added, :device do
+      arg :team_id, :string
```

Comment:
hm yeah i think we need to find a way to validate that the current token has access to subscribe to a given channel based on the token's current team claim.